src/index.ts

```diff
@@ -24,6 +25,19 @@ export const run = async () => {
         native: 'app'
       }
     });
+    
+    //Check the passed flags when installing Nightwatch and then suggest when undefined flag is been found.
+    if (argv[0] !== undefined) {
+      if ((argv[0] !== '--mobile') && (String(argv[0]) !== '--generate-config')) {
```

Comment:
@olawoyinsamson You should use the `options` variable here instead of directly using `argv`. See `minimist` [documentation](https://www.npmjs.com/package/minimist) for the difference between `args` and `options` here.

Also, your implementation is wrong, if you try running the tool locally with `npm run create-nightwatch -- --mobile --generate-config`, it will exit even when the flags passed are correct. Plus, you should not check for the flags like this (think in the lines of if we want to add more flags in the tool, how should we do it; we shouldn't need to make changes at multiple places throughout the code while doing so).

An implementation of what we're trying to achieve (give suggestions in case wrong flags are passed) is already present below in this file itself, so you should ideally make changes to that to use `didYouMean` instead of `suggestSimilarOption`, and exit the process in case wrong flags (or even arguments) are passed to the tool.

Reply:
@garg3133 Thank you for your review, i have now re-implemented the issue #85 fix as you suggested above, kindly help review again when you have a moment.